integration/shared/global/marketplace_command_test.go

```diff
@@ -0,0 +1,156 @@
+package global
```

Comment:
I know this test is modifying global state because the output of the global `cf marketplace` command would change when this test is running, but global package tests make CI and dev-loops way slower, and we try to aggressively move them into the isolated suite whenever remotely possible.

In this case, as long as all integration tests that run `cf marketplace` don't do it in a way that counts lines of output (or something similar), I believe this *should* be ok in isolated. What do you think?

Reply:
@georgi-lozev, Yep, makes perfect sense.